background.js

```diff
@@ -712,6 +714,22 @@ function swapTab() {
       });
   }
 
+  function getHoroscope(sign) {
+      var url = "http://horoscope-api.herokuapp.com/horoscope/today/"+sign;
+      var linkUrl = "https://www.ganeshaspeaks.com/horoscopes/daily-horoscope/"+sign;
+      var responseText = "Today's horoscope for "+sign+", ";
+     $.getJSON(url, function(data) {
+          setResponse(responseText+data.horoscope);
+          chrome.tabs.create({
+              'url': linkUrl
```

Comment:
I think this should be 'url' :url

Its always opening the ganesha speaks website

Reply:
I've tested it and it seems to be working fine on my end; and that is the
page that is supposed to open. The other URL is just the JSON file, and the
ganesha speaks website has more information, so users would find that more
appealing as opposed to seeing the json file.